### DISCUSSION TITLE
Matrix and XMPP room

### DISCUSSION THREAD

**Discussion by realroot2185** [FyroxEngine/Fyrox#618] [General]
Since this is a Open source project I think it would be nice to have a room there.
And it could be bridged potentially.

**Comment by mrDIMAS** [comment]
There's already a Discord server with 1000+ people in it, I don't think it is worth creating another community 🤔

**Reply by realroot2185** [reply]
https://drewdevault.com/2021/12/28/Dont-use-Discord-for-FOSS.html
